### DISCUSSION TITLE
Duda input enunciado parte 2

### DISCUSSION THREAD

**Discussion by SSParzival** [IIC2133-PUC/2024-2#145] [Tarea 2] [answered]
Hola muy buenas, estaba leyendo el enunciado y en la parte de input de la parte 2 hablan de una capacidad de la tienda; no sé a qué se refiere esto. También se me hace raro la notoria reducción en la calidad gráfica en ciertas partes del enunciado de la parte 2; me perturban con alevosía.

**Comment by 4n1t4** [comment, marked as answer]
Hola ! error de tipeo, el input debería indicar lo siguiente:
![input](https://github.com/user-attachments/assets/9c6cf4ff-bc03-4502-9d2c-bf936c2e9c76)
